src/pybind/mgr/dashboard/module.py

```diff
@@ -461,6 +462,10 @@ def set_rgw_credentials(self):
 
     def handle_command(self, inbuf, cmd):
         # pylint: disable=too-many-return-statements
+        if cmd['prefix'] == 'dashboard set-grafana-api-url':
```

Comment:
I would like to see that code in the [handle_option_command](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/settings.py#L226) because it is more logical to be there IMO.